string planner_id
string group_name
